dags/sinopia.py

```diff
@@ -46,18 +46,18 @@ def GitRdf2Marc() -> BashSensor:
     fi
     """
     return BashSensor(
-      task_id="git_rdf2marc",
-      bash_command=sensor_rdf2marc_cmd,
-      poke_interval=60,
-      timeout=60*50
+        task_id="git_rdf2marc",
+        bash_command=sensor_rdf2marc_cmd,
+        poke_interval=60,
+        timeout=60 * 50,
     )
 
+
 def Rdf2Marc(**kwargs) -> BashOperator:
     """Runs rdf2marc on a BF Instance URL"""
-    instance_url = kwargs.get('instance_url')
+    instance_url = kwargs.get("instance_url")
     if instance_url is None:
         raise ValueError("Missing Instance URL")
     return BashOperator(
-        task_id="rdf2marc",
-        bash_command=f"./exe/rdf2marc {instance_url}"
+        task_id="rdf2marc", bash_command=f"./exe/rdf2marc {instance_url}"
```

Comment:
not something that this PR changed, but is this a spot that could use some input sanitization?  looking into rdf2marc a bit, i couldn't quite tell if that was happening/needed there.  is it possible for `instance_url` be arbitrary user input?